dashboard/app/models/featured_project.rb

```diff
@@ -7,6 +7,7 @@
 #  featured_at    :datetime
 #  unfeatured_at  :datetime
 #  topic          :string(255)
+#  status         :string(255)      default("saved")
```

Comment:
Will we ever want to do something like "return all projects with status = saved"? If so, we probably want an index on it, like we have on topic

Reply:
Thanks. Do I just add the following at line 16?
```
index_featured_projects_on_status           (status)
```

Reply:
No, you need to update your migration to add an index, something like this:

https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/db/migrate/20210126021131_add_topic_to_featured_projects.rb

I believe you can rerun it by doing something like `bundle exec rake db migrate:down` then `bundle exec rails db migrate` again.

I'm not sure about the character set step though in this migration, that's a new one to me...looks like newer migrations aren't doing that, so I think you can ignore.